src/admin-views/settings/featured/title.php

```diff
@@ -0,0 +1,16 @@
+<?
+/**
+ * Title for featured settings box.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this           Template object.
```

Comment:
Here it's not an instance of `Tribe__Tickets__Admin__Views` it's an instance of `Tribe__Template`